### PR TITLE
Update vendor of containers/storage

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@mtrmac not sure what is breaking here?

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/BurntSushi/toml v1.3.2
 	github.com/containers/libtrust v0.0.0-20230121012942-c1716e8a8d01
 	github.com/containers/ocicrypt v1.1.7
-	github.com/containers/storage v1.48.0
+	github.com/containers/storage 4a3a3019d7659029f50c913c40e0bd94fc9ba284
```
Comment (flouthoc) [inline go.mod:10]: I think this should be and maybe followed by `go mod tidy`

```suggestion
	github.com/containers/storage v1.48.1-0.20230721123825-4a3a3019d765
```

**Comment by rhatdan** [comment]
FIxed

**Comment by mtrmac** [comment]
Is there a specific reason to update, on the main brach, to an unreleased version?

To be clear, I’m not at all saying that this is undesirable, I’m just curious what is the context is. Are some new users of the new features coming?

**Review by mtrmac — APPROVED** [review]

**Comment by rhatdan** [comment]
I am trying to roll everything forward, in order to get continuous testing and updates via renovate,

**Comment by rhatdan** [comment]
There seems to be something that is locking Podman to the released version of containers/image and make vendor keeps pushing it back.

**Comment by mtrmac** [comment]
> There seems to be something that is locking Podman to the released version of containers/image and make vendor keeps pushing it back.

That’s because the last released version of c/image is on a branch. The fix for that is, IIRC:
- In c/image, merge the `release-5.26` branch into `main` (so that `main` is visible in Git as a successor to the last released version)
- In a consumer, update c/image at least to that merge point.

See https://github.com/containers/image/pull/1842 and https://github.com/containers/skopeo/pull/1911 .

**Pull request merged**